src/models/hotspot_status.rs

```diff
@@ -0,0 +1,11 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Serialize, Deserialize, Debug)]
+pub struct Status {
+    /// The online status of the hotspot
+    pub online: Option<String>,
+    /// The listening IP address of the hotspot
+    pub listen_addrs: Option<Vec<String>>,
+    /// The height of hotspot
+    pub height: Option<u64>,
```

Comment:
Add the timestamp the status. Height and other information is based on when the hotspot sent out its gossip data so you need to represent height at the right time

Reply:
I have added timestamp as part of Status struct.